types/httpsnippet/index.d.ts

```diff
@@ -0,0 +1,57 @@
+// Type definitions for httpsnippet 1.23
+// Project: https://github.com/Mashape/httpsnippet
+// Definitions by: Marcell Toth <https://github.com/me>
```

Comment:
<!--
  1 failure:  This line should ...
  0 warning: 
  
  
  DangerID: danger-id-Danger;
  File: types/httpsnippet/index.d.ts;
  Line: 3;
-->
- :no_entry_sign: This line should have your github username in it, not /me